### DISCUSSION TITLE
Having a new issue with Swiper and the Slider Image Gallery component.

### DISCUSSION THREAD

**Discussion by munts** [flyntwp/flynt#519] [Q&A]
I have added the SliderImageGallery component on a few sites now (one in the v1.4 and now one in the v2.0 versions of FlyntWP) and I am now seeing the same issue on both.  That issue is that, as soon as I change the js setting of 'centeredSlides: true,' to 'centeredSlides: false,' and or add the 'loop: true,' setting the thumbnail navigation no longer works correctly.  I initially discovered this in one of my v1.4 sites, but forgot about it until I just got a request to have the thumbnails match the width of the slider.  So, I changed that one setting of 'centeredSlides' to 'false' and did some css..., which put the thumbs where I wanted, but now if you click through the thumbs to change the slides, it breaks after the 5th slide and jumps to the end, not allowing me to click the thumbs any more.  If I use the next and previous arrows to navigate the slides, the active slide no longer matches the active thumbnail...  I tried changing slideToClickedSlide: true, to false and messed with a few other settings and nothing helped. 

I then went to the swiper.js site to review the documentation and check for known issues and it appears that others are reporting similar issues.  So, I am wondering if anyone has had similar issues and if they have found a workaround?  thanks, Scott

**Comment by MadFlip** [comment]
Hi @munts, what Swiper version do you use? 
Maybe you can try to use the 'thumbs' option instead of the controller? Here is an example
[Swiper Thumbs Loop](https://codepen.io/MadFlip/pen/QWYvQLB)

**Reply by munts** [reply]
Thanks for the suggestion, I tried importing the Thumbs module and changing:
controller: {
      control: swiperSliderThumb
    }

to

thumbs: {
      swiper: swiperSliderThumb,
   }

but the thumbnail navigation failed to work if I changed centeredSlides: true, to centeredSlides: false,

Again, thanks for the suggestion.

Scott
